app/schemas/response_schema.py

```diff
@@ -0,0 +1,10 @@
+from marshmallow import Schema as SimpleSchema
+from marshmallow_jsonapi import fields
```

Comment:
import fields from marshmallow and remove dependecy for marshmallow_jsonapi

Reply:
> import fields from marshmallow and remove dependecy for marshmallow_jsonapi

Done